README.md

```diff
@@ -52,15 +52,15 @@ Syncfusion Essential XlsIO supports .xls, .xlsx, .xlsm, .xlsb, .xltm, .xltx, .xl
 
 ## License
 
-This is a commercial product and requires a paid license for possession or use. Syncfusion�s licensed software, including this component, is subject to the terms and conditions of [Syncfusion's EULA](https://www.syncfusion.com/eula/es/?utm_source=github&utm_medium=listing&utm_campaign=github-xlsio-examples). You can purchase a licnense [here](https://www.syncfusion.com/sales/products?utm_source=github&utm_medium=listing&utm_campaign=github-xlsio-examples) or start a free 30-day trial [here](https://www.syncfusion.com/account/manage-trials/start-trials?utm_source=github&utm_medium=listing&utm_campaign=github-xlsio-examples).
+This is a commercial product and requires a paid license for possession or use. Syncfusion�s licensed software, including this component, is subject to the terms and conditions of [Syncfusion's EULA](https://www.syncfusion.com/eula/es/?utm_source=github&utm_medium=listing&utm_campaign=github-xlsio-examples). You can purchase a licnense [here](https://www.syncfusion.com/sales/products?utm_source=github&utm_medium=listing&utm_campaign=github-xlsio-examples) or start a free 30-day trial [here](https://www.syncfusion.com/account/manage-trials/start-trials?utm_source=github&utm_medium=listing&utm_campaign=github-xlsio-examples).
```

Comment:
check the special characters
<img width="982" alt="image" src="https://user-images.githubusercontent.com/61148953/207626900-cce768df-dd08-4f95-bec7-9ce107be24a9.png">